.github/workflows/ai-lab-e2e-nightly-windows.yaml

```diff
@@ -81,15 +76,16 @@ on:
           - Standard_D8s_v4
           - Standard_E8as_v5
           - Standard_E4as_v5
+      mapt_params:
+        default: 'IMAGE=quay.io/redhat-developer/mapt,VERSION_TAG=v0.9.3,EXCLUDED_REGIONS=westindia,centralindia,southindia,australiacentral,australiacentral2,australiaeast,australiasoutheast,southafricanorth,southafricawest'
```

Comment:
you probably need to wrap regions into double quotes.

Reply:
good point